src/mame/drivers/raiden_ms.cpp

```diff
@@ -658,26 +669,27 @@ void raiden_ms_state::raidenm(machine_config &config)
 
 	GFXDECODE(config, "gfxdecode", "palette", gfx_raiden_ms);
 
-	GENERIC_LATCH_8(config, m_soundlatch);
+	GENERIC_LATCH_8(config, m_soundlatch[0]);
+	GENERIC_LATCH_8(config, m_soundlatch[1]);
 
 	/* sound hardware */
 	SPEAKER(config, "mono").front_center();
 	YM2203(config, m_ym1, XTAL(4'000'000)/4); // unknown clock
 	m_ym1->add_route(0, "mono", 0.15);
 	m_ym1->add_route(1, "mono", 0.15);
 	m_ym1->add_route(2, "mono", 0.15);
-	m_ym1->add_route(3, "mono", 0.10);
+	m_ym1->add_route(3, "mono", 0.40);
 
 	YM2203(config, m_ym2, XTAL(4'000'000)/4); // unknown clock
 	m_ym2->add_route(0, "mono", 0.15);
 	m_ym2->add_route(1, "mono", 0.15);
 	m_ym2->add_route(2, "mono", 0.15);
-	m_ym2->add_route(3, "mono", 0.10);
+	m_ym2->add_route(3, "mono", 0.40);
 
 	MSM5205(config, m_msm, XTAL(384'000)); // unknown clock
 	m_msm->vck_legacy_callback().set(FUNC(raiden_ms_state::adpcm_int));
-	m_msm->set_prescaler_selector(msm5205_device::S48_4B); // unverified
-	m_msm->add_route(ALL_OUTPUTS, "mono", 0.25);
+	m_msm->set_prescaler_selector(msm5205_device::S96_4B); // unverified
+	m_msm->add_route(ALL_OUTPUTS, "mono", 1.00);
```

Comment:
If you run a debug build, do you hit the limiter/compressor on peaks when you boost the outputs like this?

Reply:
`-speaker_report` didn't show any clipping during gameplay.